File: FLUTTER/igest_app/lib/models/principal/direccion.dart
class Direccion {
  int idDireccion;
  bool corrido;
  int idTipoVia;
  String nomTipoVia;
  String nomVia;
  int idTipoZona;
  String nomTipoZona;
  String nomZona;
  String referencia;
  int idPais;
  String idDistrito;
  String latitud;
  String longitud;
  int idEmpresa;

  Direccion({
    required this.idDireccion,
    required this.corrido,
    required this.idTipoVia,
    required this.nomTipoVia,
    required this.nomVia,
    required this.idTipoZona,
    required this.nomTipoZona,
    required this.nomZona,
    required this.referencia,
    required this.idPais,
    required this.idDistrito,
    required this.latitud,
    required this.longitud,
    required this.idEmpresa,
  });

  factory Direccion.fromJson(Map<String, dynamic> json) {
    return Direccion(
      idDireccion: json['id_direccion'],
      corrido: json['corrido'],
      idTipoVia: json['id_tipo_via'],
      nomTipoVia: json['nom_tipo_via'],
      nomVia: json['nom_via'],
      idTipoZona: json['id_tipo_zona'],
      nomTipoZona: json['nom_tipo_zona'],
      nomZona: json['nom_zona'],
      referencia: json['referencia'],
      idPais: json['id_pais'],
      idDistrito: json['id_distrito'],
      latitud: json['latitud'],
      longitud: json['longitud'],
      idEmpresa: json['id_empresa'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id_direccion': idDireccion,
      'corrido': corrido,
      'id_tipo_via': idTipoVia,
      'nom_tipo_via': nomTipoVia,
      'nom_via': nomVia,
      'id_tipo_zona': idTipoZona,
      'nom_tipo_zona': nomTipoZona,
      'nom_zona': nomZona,
      'referencia': referencia,
      'id_pais': idPais,
      'id_distrito': idDistrito,
      'latitud': latitud,
      'longitud': longitud,
      'id_empresa': idEmpresa,
    };
  }
}

File: NET/_XXX_com.yupanastudio.polariss.app401.model/admin/aaa/Direccion.cs
using System.Linq;

namespace com.yupanastudio.polariss.app
{
    public class Direccion
    {
        public long idDireccion { get; set; }
        public int idTipoVia { get; set; }
        public string nomTipoVia { get; set; }
        public string nomVia { get; set; }
        public int idTipoZona { get; set; }
        public string nomTipoZona { get; set; }
        public string nomZona { get; set; }
        public string referencia { get; set; }
        public int idPais { get; set; }
        public string idDistrito { get; set; }
        public string latitud { get; set; }
        public string longitud { get; set; }
        public int idEmpresa { get; set; }
        public int countUsoX { get; set; }
        public string fullDirX
        {
            get
            {
                return
                    nomTipoVia + " " + nomVia + (idTipoZona == 132 ? "" : "; " + nomTipoZona + " " + nomZona) + "; " +
                    Vars.lsDists.Where(x => x.idDistrito == idDistrito).FirstOrDefault().fullNameX_DisProDep;
            }
        }

        public Direccion() { }

        /// <summary>
        /// Para crear un nuevo objeto y no se enlace por referencia
        /// </summary>
        public Direccion(Direccion obDir)
        {
            this.idDireccion = obDir.idDireccion;
            this.idTipoVia = obDir.idTipoVia;
            this.nomTipoVia = obDir.nomTipoVia;
            this.nomVia = obDir.nomVia;
            this.idTipoZona = obDir.idTipoZona;
            this.nomTipoZona = obDir.nomTipoZona;
            this.nomZona = obDir.nomZona;
            this.referencia = obDir.referencia;
            this.idPais = obDir.idPais;
            this.idDistrito = obDir.idDistrito;
            this.latitud = obDir.latitud;
            this.longitud = obDir.longitud;
            this.idEmpresa = obDir.idEmpresa;
            this.countUsoX = obDir.countUsoX;
        }

    }//end class

}
